### PR TITLE
Task/mqtt

### PR DESCRIPTION
**Pull request by XavierVal**
Added test cases for mqtt feature

### CODE REVIEW

**Comment by gsevillano** [comment]
LGTM

File: tests/steps/happy_path.py [hunk 1]
```diff
@@ -764,3 +781,59 @@ def step_impl(context, DEVICE_ID):
...
+@step("a service and subservice are provisioned")
+def service_subservice_default_provision(context):
+    """
+    :type context: behave.runner.Context
+    """
+    json_data_s = """{
+                                    "DOMAIN_ADMIN_USER": "cloud_admin",
+                                    "NEW_SERVICE_ADMIN_PASSWORD": "4passw0rd",
+                                    "KEYPASS_PORT": "8080",
+                                    "KEYSTONE_PORT": "5000",
```
Comment (AlvaroVega) [inline tests/steps/happy_path.py]: Are those ports right ?

**Comment by AlvaroVega** [comment]
LGTM

File: tests/steps/happy_path.py [hunk 2]
```diff
@@ -764,3 +781,59 @@ def step_impl(context, DEVICE_ID):
...
+
+@step("a service and subservice are provisioned")
+def service_subservice_default_provision(context):
+    """
+    :type context: behave.runner.Context
+    """
+    json_data_s = """{
+                                    "DOMAIN_ADMIN_USER": "cloud_admin",
+                                    "NEW_SERVICE_ADMIN_PASSWORD": "4passw0rd",
+                                    "KEYPASS_PORT": "8080",
```
Comment (AlvaroVega) [inline tests/steps/happy_path.py]: If you a re using ORC api then KEYPASS_PORT, KEYSTONE_PORT, KEYSPASS_HOST has no sense

File: tests/properties.json [hunk 3]
```diff
@@ -117,12 +139,27 @@
...
+        "user_service": "admin_domain",
         "user_subservice": "/sub1",
         "user_role": "service_customer"
+      },
+      "user_3": {
+        "user_name": "admin_bb",
+        "user_password": "password",
+        "user_service": "whatever",
+        "user_subservice": "/tt",
+        "user_role": "service_customer"
```
Comment (anabelengp) [inline tests/properties.json]: shouldn't admin_bb have role `admin`?

Comment (XavierVal) [inline tests/properties.json]: Fixed in 35d76f8

**Comment by anabelengp** [comment]
LGTM

**Pull request merged**